go-controller/pkg/clustermanager/userdefinednetwork/template/net-attach-def-template_test.go

```diff
@@ -183,6 +183,19 @@ var _ = Describe("NetAttachDefTemplate", func() {
 			&udnv1.UserDefinedNetwork{Spec: udnv1.UserDefinedNetworkSpec{
 				Topology: udnv1.NetworkTopologyLayer3, Layer2: &udnv1.Layer2Config{}}},
 		),
+		Entry("UDN, invalid IPAM lifecycle",
```

Comment:
maybe be a bit more clear here: AFAIU, the IPAM lifecycle is invalid *because* IPAM is disabled.